roku_dev_cli/version.py

```diff
@@ -1 +1 @@
-__version__ = '0.32.1'
+__version__ = '0.32.2'
```

Comment:
Given that we're adding new CLI arguments, we should probably make this a minor-version bump] to follow semver guidance: https://semver.org/#summary

Reply:
done